pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -887,7 +884,7 @@ func (adc *attachDetachController) GetServiceAccountTokenFunc() func(_, _ string
 
 func (adc *attachDetachController) DeleteServiceAccountTokenFunc() func(types.UID) {
 	return func(types.UID) {
-		klog.Errorf("DeleteServiceAccountToken unsupported in attachDetachController")
+		klog.ErrorS(nil, "DeleteServiceAccountToken unsupported in attachDetachController")
```

Comment:
Would it be possible to pass a logger into `DeleteServiceAccountTokenFunc` and use that instead of `klog.ErrorS`?

Reply:
This associates many plugins and many files in different places
I think it's not worth it just to print this log

Reply:
Makes sense.